desc/objectives/__init__.py

```diff
@@ -55,9 +62,3 @@
     FixThetaSFL,
 )
 from .objective_funs import ObjectiveFunction
-from .utils import (
```

Comment:
Why not include the utils in the init file?

Reply:
The stuff left in utils is not meant to be public, so its not part of the module init.